doc/reference/cardano-node-cli-reference.md

```diff
@@ -5,7 +5,7 @@ The command line interface (CLI) provides a collection of tools for generating k
 This section provides a reference of the core `cardano-cli` commands and their associated subcommands:
 
 *cardano-cli*
-The set of `cardano-cli` commands include:
+The set of `cardano-cli` commands includes:
```

Comment:
I think `include` is the right grammar here.

Reply:
No, 'The set... includes' is correct.